core-plugins/src/main/java/io/cdap/plugin/transform/ProjectionTransform.java

```diff
@@ -176,16 +185,18 @@ private void init(Schema inputSchema) {
           }
         }
         if (containAllFields) {
-          throw new IllegalArgumentException("'Fields to drop' cannot contain all the fields of input s chema.");
+          collector.addFailure("'Fields to drop' cannot contain all the fields of input schema.",
+              null).withConfigProperty(DROP);
```

Comment:
style: move `null)` to the previous line, and use 2 space indentation instead of 4.

Reply:
Fixed